lib/version.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lib
+
+var Version = "0.0.0+was-not-built-correctly"
```

Comment:
Oh wow, I completely missed that. Will do.

On Wed, Oct 21, 2015 at 11:48 AM, Jonathan Boulle <notifications@github.com>
wrote:

> In lib/version.go
> <https://github.com/appc/acbuild/pull/45#discussion_r42665596>:
>
> > +//
> > +// Licensed under the Apache License, Version 2.0 (the "License");
> > +// you may not use this file except in compliance with the License.
> > +// You may obtain a copy of the License at
> > +//
> > +//     http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +package lib
> > +
> > +var Version = "0.0.0+was-not-built-correctly"
>
> Use AppContainerVersion. c.f.
> https://github.com/coreos/rkt/blob/9ddb77f0fbc8d5cabbd5a70b7873882341d87ce7/rkt/version.go#L38
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/acbuild/pull/45/files#r42665596>.
>